account-gui/src/locale/en.js

```diff
@@ -327,9 +327,9 @@ I18n.translations.en = {
         no: "I refuse",
         yesLink: "/proceed",
         yes: "Proceed",
-        warning: "Login without the eduApp is strongly discouraged. The service {{service}} will not receive your attributes.",
-        warningTitle: "Get the eduApp",
-        confirmLabel: "Get the eduApp",
+        warning: "Login without the eduID app is strongly discouraged. You will not be able to log in to {{service}}.",
```

Comment:
It's "discouraged" but then "does not work"? So it's not discouraged but just simply not allowed?